packages/apps/human-app/server/src/common/guards/jwt.auth.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Reflector } from '@nestjs/core';
+import { AuthGuard } from '@nestjs/passport';
+import { JwtUserData } from '../utils/jwt-token.model';
+
+@Injectable()
+export class JwtAuthGuard extends AuthGuard('jwt-http') implements CanActivate {
+  constructor(private readonly reflector: Reflector) {
+    super();
+  }
+
+  public async canActivate(context: ExecutionContext): Promise<boolean> {
+    // Check for public routes first
+    const isPublic = this.reflector.getAllAndOverride<boolean>('isPublic', [
```

Comment:
We have this check, but don't have such decorator/metadata.
Let's define this decorator and use it in `getAllAndOverride` (see RepO for example)